### PR TITLE
Custom export generator for easier treeShaking

### PR DESCRIPTION
**Pull request by karelhala**
Output of file mapper
```
{
    "CriticalBattery": "Battery",
    "HighBattery": "Battery",
    "MediumBattery": "Battery",
    "LowBattery": "Battery",
    "NullBattery": "Battery",
    "ConnectedBreadcrumbs": "Breadcrumbs",
    "ConditionalFilterType": "ConditionalFilter",
    "groupType": "ConditionalFilter",
    "DarkContext": "Dark",
    "FilterDropdown": "Filters",
    "FilterInput": "Filters",
    "LabeledInput": "Input",
    "PageHeaderTitle": "PageHeader",
    "dropDirection": "Pagination",
    "SkeletonSize": "Skeleton",
    "SortDirection": "Table",
    "TableVariant": "Table",
    "TableHeader": "Table",
    "TableBody": "Table",
    "TableFooter": "Table"
}
```

Ok the scope of this PR got a bit bigger, now it also produces mapped file for notifications and adds direct imports for icons in notifications and components.

### CODE REVIEW

File: packages/components/config/generateMapper.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        const data = generateData(await readFile(filePath));
+
+        return data.filter(item => item !== exportName).reduce((acc, curr) => ({
+            ...acc,
+            [curr]: exportName
+        }), {});
+    }));
+
+    writeMapper(mappedFiles);
+    moveCustomName();
```
Comment (Hyperkid123) [inline packages/components/config/generateMapper.js]: I don't see this function defined anywhere. Is it a global?

Comment (karelhala) [reply packages/components/config/generateMapper.js]: Yeah, this function was removed.

File: packages/components/src/Components/DownloadButton/DownloadButton.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ class DownloadButton extends Component {
                         iconComponent={null}
                         onToggle={ this.onToggle }
                     >
-                        <ExportIcon size={ IconSize.sm } />
+                        <ExportIcon size="sm" />
```
Comment (AllenBW) [inline packages/components/src/Components/DownloadButton/DownloadButton.js]: So what's the convention, single quotes or double quotes? I see a lotta each of em used (or is it kinda 🤷‍♂ )

Comment (karelhala) [reply packages/components/src/Components/DownloadButton/DownloadButton.js]: Double quotes for html-like and single quotes for JS

**Pull request closed without merging**